src/components/Comment/Comment.scss

```
@@ -0,0 +1,35 @@
+.comment {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  box-shadow: 0 0 10px rgba(197, 197, 197, 0.5); /* Параметры тени */
```

Try to comment your code using English
```suggestion
  box-shadow: 0 0 10px rgba(197, 197, 197, 0.5); /* Shadow params */
```

---

fixed